config/gazebo9.upstream.yaml

```diff
@@ -0,0 +1,6 @@
+name: gazebo9_import
```

Comment:
should this name and the file have the platform name as well ? (like you did for stretch below)

Reply:
I don't have a strong feeling either way.  The existing configs have a bunch of styles.  I guess more information is better, so I'll change it (and the name of the file) to be more clear.